src/lib/transition_frontier_controller_tests/test_transition_frontier_persistence.ml

```diff
@@ -50,10 +50,14 @@ let%test_module "Transition Frontier Persistence" =
       Monitor.try_with (fun () ->
           let frontier_persistence = create_persistence ~directory_name in
           let%bind result = f (frontier, frontier_persistence) in
-          let%map () =
-            Transition_frontier_persistence.close_and_finish_copy
-              frontier_persistence
+          frontier_persistence.is_accepting_diffs <- false ;
```

Comment:
I think it is bad style to have a public mutable field that needs to be set in this fashion. It's better to encapsulate the mutable information inside the other module and provide a clear interface for interacting with the mutation. See my other comment first, though, as I think you should consider getting rid of this mutation all together.